include/zephyr/drivers/pwm.h

```diff
@@ -925,6 +925,18 @@ static inline int pwm_capture_nsec(const struct device *dev, uint32_t channel,
 	return 0;
 }
 
+/**
+ * @brief Validate that the PWM device is ready.
+ *
+ * @param spec PWM specification from devicetree
+ *
+ * @retval true if the PWM device is ready for use and false otherwise.
```

Comment:
```suggestion
 * @retval true If the PWM device is ready for use
 * @retval false If the PWM device is not ready for use
```

^ retval usage was incorrect

Reply:
Thanks, fixed